### DISCUSSION TITLE
How to set the width of div elements inside a div?

### DISCUSSION THREAD

**Discussion by jcmoyongo** [tailwindlabs/tailwindcss#10965] [Help]
Hello all,

Very new to Tailwind CSS or CSS. I am trying my best to format 4 divs inside a div. I want the first two to have the same width, the third one another width (a bid smaller) and the fourth one to take up the remaining space. The embedded image is an illustration of what I am trying to achieve, just in a different order:
![image](https://user-images.githubusercontent.com/7173987/230618004-cb726cd7-475d-4183-887d-62eae71ab07f.png)

This is what my rendering looks like so far after 3 days of trying:
![image](https://user-images.githubusercontent.com/7173987/230618166-1d138eda-3f1b-4f3f-810f-81e9e38fabf3.png)

Below is my React JS + CSS code:

```jsx
<div className="flex flex-row flex-wrap rounded white-glassmorphism p-3 m-2 cursor-pointer w-full hover:shadow-xl">
  <div className="grid grid-cols-4 divide-x">
    <div class="flex items-center">
      <p>{game.Home}</p>
      <div
        className={`w-10 h-10 rounded-full flex justify-center items-center ${"bg-[#2952E3]"}`}
      >
        {<MdSportsBasketball fontSize={21} className="text-white" />}
      </div>
    </div>
    <div class="flex items-center">
      <div
        className={`w-10 h-10 rounded-full flex justify-center items-center ${"bg-[#2952E3]"}`}
      >
        {<MdSportsBasketball fontSize={21} className="text-white" />}
      </div>
      <p>{game.Away}</p>
    </div>
    <div>
      <p>at {game.Time}</p>
    </div>
    <div>
      <p>at {game.Venue}</p>
    </div>
  </div>
</div>;

```

Thanks in advance for the help!

**Comment by wongjn** [comment]
Set the grid column track sizes to your needs by applying a custom `grid-template-columns` value, for example, `10rem 10rem 8rem 1fr`: https://play.tailwindcss.com/9EfubE45Ym.

**Reply by jcmoyongo** [reply]
Thanks @wongjn ...You've pointed me in the right direction. I am having trouble with the last two columns not wrapping properly when I shrink the form horizontally...The last two columns' text stays outside of the window. I'll try to fix it and share an image, no matter the outcome.
